server/v1/helpers/queryHelper.js

```diff
@@ -17,7 +17,13 @@ export default {
       { model: User, as: 'author', attributes: ['username', 'firstName', 'lastName', 'bio', 'image'] },
       { model: Rating, as: 'ratings', attributes: ['id', 'userId', 'value', 'createdAt', 'updatedAt'] },
       { model: Tag, as: 'tags', attributes: ['id', 'articleId', 'title', 'createdAt', 'updatedAt'] },
-      { model: Comment, as: 'comments', attributes: ['id', 'body', 'createdAt', 'updatedAt'] },
+      {
+        model: Comment,
+        as: 'comments',
+        include: [
+          { model: User, as: 'commenter' },
```

Comment:
NIce work including the commenter as it would make displaying the data on the client much easier.